fp.js

```diff
@@ -1,28 +1,31 @@
-// Convert the following code to use `.map`
+// // Convert the following code to use `.map`
 
-var peoplesAges = []
-var people = [
-  {name: "Jane", age: 32},
-  {name: "Jessica", age: 34},
-  {name: "Jasmine", age: 39}
-]
-people.forEach(function(person){
-  peoplesAges.push(person.age)
-});
+// var peoplesAges = []
+// var people = [
+//   {name: "Jane", age: 32},
+//   {name: "Jessica", age: 34},
+//   {name: "Jasmine", age: 39}
+// ]
+// people.map(function(person){
```

Comment:
No need to `push` in this one. Instead, you can save the result of `.map` directly to a variable. Inside the `.map` callback, make sure to return whatever value you want to save, like so...

```js
let ages = people.map(person => {
  return person.age
})
```